packages/nocodb/src/lib/models/Audit.ts

```diff
@@ -31,37 +31,49 @@ export default class Audit implements AuditType {
     return audit && new Audit(audit);
   }
 
+  // Will only await for Audit insertion if `forceAwait` is true, which will be true in test environment by default
   public static async insert(
     audit: Partial<
       Audit & {
         created_at?;
         updated_at?;
       }
     >,
-    ncMeta = Noco.ncMeta
+    ncMeta = Noco.ncMeta,
+    { forceAwait }: { forceAwait: boolean } = {
```

Comment:
could be just  `forceAwait = process.env['TEST'] === 'true'`

Reply:
@pranavxc I have kept it for future use. Since its an optional param I thought it won't hurt to keep it here.